strings/DOOMspeak.json

```diff
@@ -0,0 +1,26 @@
+{
+
+	"DOOMspeak": {
+		"antags": "Rapp Snitches",
+		"attack him": "And they say he accosted the man with the sliced wig",
+		"attacked her": "And they say he accosted the man with the sliced wig",
+		"attacked them": "And they say he accosted the man with the sliced wig",
+		"bad": "So nasty that it's probably somewhat of a travesty",
+		"fire": "fire vocaled With ash and molten glass like Eyjafjallajökull",
+		"for": "pour",
+		"gibber": "Meat Grinder",
+		"give it back": "Matter fact, gimme back my bracelet and my Shearling",
+		"hi": "yo, It's Victor",
+		"husk": "little charred body smelted like chicken",
+		"me": "the MADVILLIAN",
+		"murderboning": "dashing in a beat-rhyme crime spree",
+		"my name": "the mad name",
+		"retard": "average bug-a-boo",
```

Comment:
im not sure if this fits ToS

Reply:
also doesnt make sense, since bug-a-boo means object of fear, not sure what the relation is

Reply:
bug-a-boo is something that causes more trouble than it's worth

Reply:
and wouldnt that....... be considered highly offensive? i dont think this line is needed when having it causes offense for no lore related reasons

Reply:
it's "retard" floyd is talking about, in case either of you didnt realize

Reply:
> and wouldnt that....... be considered highly offensive? i dont think this line is needed when having it causes offense for no lore related reasons
I wasn't aware ss13 was intended to be politically correct. I guess we gotta put in a pull request to remove the italian mustache and the french beret, too

Reply:
bro noone cares just follow github's ToS you nonce.

Reply:
> 
> 
> > and wouldnt that....... be considered highly offensive? i dont think this line is needed when having it causes offense for no lore related reasons
> > I wasn't aware ss13 was intended to be politically correct. I guess we gotta put in a pull request to remove the italian mustache and the french beret, too

point is, bug-a-boo literally doesnt describe a "retard" ... the definition you give of "more trouble than its worth" isnt an issue of political correctness (though we still have to follow githubs ToS), but its also offensive for the sole sake to cause offense, causing offense is fine if it makes sense, this doesnt, its just unnesscary